Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "lang"
```

Comment:
given the lack of namespaces I suggest something tiresome and long, e.g. google-fonts-languages